src/tests/unit/doughs-serializer.test.js

```diff
@@ -99,6 +118,21 @@ describe('test doughs serializer', () => {
     describe('when it gets a dough object array with valid formatting', () => {
       initializeSerializeDoughsTest([emptyStringSpecialInstructionsDough]);
 
+      it('converts integer fields from strings', () => {
+        [
+          'gramsFlour',
+          'gramsWater',
+          'waterTemp',
+          'gramsYeast',
+          'gramsSalt',
+          'bulkFermentTime',
+          'proofTime',
+          'gramsSugar',
+          'gramsOliveOil',
+        ].forEach((attribute) => serializedDoughs
+          .data[0].attributes[attribute].should.be.a('number'));
+      });
+
       it('returns an object with a data member that is an array', () => {
         serializedDoughs.data.should.be.a('array');
```

Comment:
I might miss it, but I think you should also add the test case to assert it will return an object for get by ID response (serializedDoughs).

Reply:
this is sort of implicitly tested for but I can make it explicit.